.gitignore

```diff
@@ -2,3 +2,4 @@ tests/Fixtures/App/var
 !tests/Fixtures/App/var/.gitempty
 composer.lock
 vendor
+.idea
```

Comment:
this is developer environment specific configuration and should go into your global git ignore configuration.